### PR TITLE
MDE: Fix error in creation of fields from objectDef

### PR DESCRIPTION
**Pull request by sukunis**
Call getChildsByType returned an empty array, but caller expected NULL if no child was found. Fixed by return NULL or not empty list.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

File: src/main/java/org/openmicroscopy/shoola/agents/fsimporter/mde/components/view/ModuleTree.java [hunk 1]
```diff
@@ -307,35 +307,48 @@ public void customizeTree(String micName){
...
-									}
-								}
+					insertCustomizeNode(entry.getKey(),entry.getValue(),cTree,confMap);
+				}
+			}
+		}
+	}
+
+	private void insertCustomizeNode(String nodeType,ModuleConfiguration nodeConf, DefaultMutableTreeNode cTree,HashMap<String,ModuleConfiguration> confMap){
+		// test auf element ist bereits child of root
```
Comment (dominikl) [inline src/main/java/org/openmicroscopy/shoola/agents/fsimporter/mde/components/view/ModuleTree.java:317]: Interesting mix of German and English :-)

**Comment by dominikl** [comment]
👍 Looks good to me.

**Pull request merged**